biorbd_optim/optimal_control_program.py

```diff
@@ -248,13 +254,25 @@ def __define_multiple_shooting_nodes_per_phase(self, nlp, idx_phase):
             else:
                 V_bounds.min[offset : offset + nlp["nu"]] = nlp["U_bounds"].min
                 V_bounds.max[offset : offset + nlp["nu"]] = nlp["U_bounds"].max
-            V_init.init[offset : offset + nlp["nu"]] = nlp["U_init"].init
+            if nlp["U_init"].init_lineaire:
+                V_init.init[offset : offset + nlp["nu"]] = [
+                    (nlp["U_init"].init[1 + 2*i] - nlp["U_init"].init[2*i]) * (k/nlp["ns"]) +
+                    nlp["U_init"].init[2*i] for i in range(nlp["nu"])
+                ]
+            else:
+                V_init.init[offset: offset + nlp["nu"]] = nlp["U_init"].init
```

Comment:
Same thing here, because this is not factor in the proper function, you have to duplicate the code. And duplicating code is evil